recipes/pypsa/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pypsa" %}
+{% set version = "0.14.1" %}
+{% set sha256 = "477716d83806a29e75bcb48c3e7c19b8f96ca1af68d7eaf6fd86202f3c79c913" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - cartopy >=0.16
+    - matplotlib
+    - networkx >=1.10
+    - numpy
+    - pandas >=0.19.0
+    - pyomo >=5.3
+    - python
+    - scipy
+    - six
+    - netcdf4
+    - pytables
+
+test:
+  imports:
+    - pypsa
+
+about:
+  home: https://pypsa.org/
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE.txt
+  summary: Python for Power Systems Analysis
+  description: |
+    PyPSA is a free software toolbox for simulating and optimising modern power
+    systems that include features such as conventional generators with unit
+    commitment, variable wind and solar generation, storage units, coupling to
+    other energy sectors, and mixed alternating and direct current networks.
+    PyPSA is designed to scale well with large networks and long time series.
+  doc_url: https://pypsa.org/doc/
+  dev_url: https://github.com/PyPSA/PyPSA
+
+extra:
+  recipe-maintainers:
+    - coroa
+    - nworbmot
```

Comment:
@nworbmot Are you happy to be listed as a maintainer of this recipe?

Reply:
Yes, absolutely! Thank you!